lib/active_record/events/naming.rb

```diff
@@ -6,10 +6,12 @@ class Naming
       def initialize(infinitive, options = {})
         @infinitive = infinitive
         @object = options[:object].presence
+        @field_type = options[:field_type]
```

Comment:
Let's add `to_sym` here to allow passing the value both as a symbol and a string.